src/ontology/foodon-edit.owl

```diff
@@ -19736,6 +19754,150 @@ AnnotationAssertion(dc:date obo:FOODON_00004287 "2022-06-15T11:40:45Z"^^xsd:date
+AnnotationAssertion(dc:date obo:FOODON_00004293 "2022-08-24T12:49:23Z"@en)
+AnnotationAssertion(rdfs:label obo:FOODON_00004293 "finisher cow"@en)
+SubClassOf(obo:FOODON_00004293 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004294 (stocker cow)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004294 "A  cow that has been weaned and is being fed in a way to add physical bulk to its body."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004294 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004294 "2022-08-24T12:53:15Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004294 "stocker cow"@en)
+SubClassOf(obo:FOODON_00004294 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004295 (weanling cow)
+
+AnnotationAssertion(dc:contributor obo:FOODON_00004295 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004295 "2022-08-24T13:39:16Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:FOODON_00004295 "A young cow that has been weaned and consumes solid food."@en)
+AnnotationAssertion(rdfs:label obo:FOODON_00004295 "weanling cow"@en)
+SubClassOf(obo:FOODON_00004295 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004296 (freemartin cow)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004296 "A cow that is genetically female, but has many characteristics of a male and is usually infertile."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004296 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004296 "2022-08-24T13:51:12Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004296 "freemartin cow"@en)
+SubClassOf(obo:FOODON_00004296 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004297 (nursing pig)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004297 "A lactating sow that feeds a piglet farrowed by another pig."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004297 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004297 "2022-08-24T14:38:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004297 "nursing pig"@en)
+SubClassOf(obo:FOODON_00004297 obo:FOODON_03411136)
+
+# Class: obo:FOODON_00004298 (poultry)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004298 "A domesticated bird that is kept by humans for its eggs, meat or feathers."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004298 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:date obo:FOODON_00004298 "2022-08-25T10:01:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004298 "poultry"@en)
+SubClassOf(obo:FOODON_00004298 obo:FOODON_00001283)
+
+# Class: obo:FOODON_00004299 (chick)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004299 "A young offspring of a bird."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004299 "https://www.merriam-webster.com/dictionary/chick"@en)
```

Comment:
Lets avoid using commercial dictionary sites if possible, due to licensing issues around content.  Using https://en.wiktionary.org/wiki/chick should be fine.

Reply:
Got it, I've changed that.